.prettierrc

```diff
@@ -0,0 +1,5 @@
+{
+  "semi": false,
+  "singleQuote": true,
+  "trailingComma": "es5"
```

Comment:
This is now the default in prettier 2.0, so I think we could update prettier and remove it? https://prettier.io/docs/en/options.html#trailing-commas